### ISSUE TITLE
empty DNSPrefix error should exit immediately

### ISSUE THREAD

**Issue by jackfrancis** [Azure/acs-engine#1404]
An empty DNS prefix value seems to generate additional, unnecessary errors.

---

**Is this an ISSUE or FEATURE REQUEST?** (choose one): ISSUE

---

**What version of acs-engine?**: v0.6.0

---

**Orchestrator and version (e.g. Kubernetes, DC/OS, Swarm)** Kubernetes

**What happened**:

```
NFO[0000] Error returned by LoadContainerService: missing Properties.MasterProfile.DNSPrefix. Attempting to load container service using LoadContainerServiceForAgentPoolOnlyCluster 
FATA[0000] error parsing the api model: Namespace Properties.DNSPrefix is not caught, Key: 'Properties.DNSPrefix' Error:Field validation for 'DNSPrefix' failed on the 'required' tag
Key: 'Properties.ServicePrincipalProfile.ClientID' Error:Field validation for 'ClientID' failed on the 'required' tag
Key: 'Properties.ServicePrincipalProfile.Secret' Error:Field validation for 'Secret' failed on the 'required' tag
Key: 'Properties.CertificateProfile' Error:Field validation for 'CertificateProfile' failed on the 'required' tag 
```

**What you expected to happen**: I expect *just* the DNSPrefix error.

**How to reproduce it** (as minimally and precisely as possible): Generate a template using an api model without a dNSPrefix value

**Comment by itowlson** [comment]
I think #1453 fixes this too.

**Comment by SaaldjorMike** [comment]
Can this be closed then since #1453 was merged?

**Comment by jackfrancis** [comment]
Yes, thanks for the nudge @SaaldjorMike. Please re-open if you can confirm that master (post merge of #1453) still exhibits this bug.
